x-pack/plugins/cases/docs/openapi/paths/api@cases@{caseid}@comments@{commentid}.yaml

```diff
@@ -0,0 +1,160 @@
+post:
```

Comment:
> post:

I think this operation should be removed from this file. Per https://www.elastic.co/guide/en/kibana/master/cases-api-add-comment.html there's no comment_id at the end of this path:

> POST <kibana host>:<port>/api/cases/<case_id>/comments

Reply:
Removed in [15f9a1c](https://github.com/elastic/kibana/pull/133400/commits/15f9a1ca39e168ad061d7ae7e6fce8dbe5e9f2be).